### DISCUSSION TITLE
Loading Brunch Framework

### DISCUSSION THREAD

**Discussion by tsarmark** [sebanc/brunch#697] [General]
Hi, maybe it's a very basic question, but how long do I have to wait in "Loading Brunch Framework"? It is that I have already been about 2 hours and it does not go beyond that screen. Thanks in advance

**Comment by brkeejp** [comment]
I think this depend your hardware (if there is no error of course :)
in your case most probably error though :P

**Comment by sebanc** [comment]
@tsarmark It is not normal for brunch to take that much time to boot. What is your laptop/pc model ?

**Reply by tsarmark** [reply]
Hi, I have a Jumper Ezbook 3pro. Is a chinease laptop :(

**Reply by sebanc** [reply]
Try the latest unstable release, if it does not boot, select brunch (debug mode) when you see grub and press "e" quickly, then replace "/kernel" by "/kernel-4.19" and press F10 to boot

**Comment by jfield9734** [comment]
If this is your first boot, you may also want to consider the USB flash drive you are using.  The first time I ran Brunch, I used a PNY 32GB drive.  I ran into this same issue where it was stuck on "Loading Brunch Framework".  Turns out this drive had issues, even after re-writing it, using different builds, etc..  I changed over to a Sandisk Ultra 16GB and it loaded within only a few minutes on first boot using the same image that was hanging on the PNY 32GB.
